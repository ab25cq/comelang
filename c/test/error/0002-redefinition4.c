/*
PATTERN: x
PATTERN: typedef
PATTERN: c:9
*/

typedef int  x;
typedef int  x;
typedef int *x;
