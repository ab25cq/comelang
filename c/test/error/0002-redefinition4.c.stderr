incompatible redefinition of typedef x at test/error/0002-redefinition4.c:9:1
typedef int *x;
^
